client/lib/two-step-authorization/index.js

```diff
@@ -71,6 +71,7 @@ TwoStepAuthorization.prototype.validateCode = function( args, callback ) {
 				userSettings.fetchSettings();
 				applicationPasswords.fetch();
 				connectedApplications.fetch();
+				profileLinks.reAuthRequired = false;
```

Comment:
I'm looking through usages. Did we never have any handlers that set this back to false before?